services/net/notification/Models/TnoRazorLightEngine.cs

```diff
@@ -0,0 +1,19 @@
+﻿using RazorLight;
+
+namespace TNO.Services.Notification.Models
+{
+    public class TnoRazorLightEngine : ITnoRazorLightEngine
+    {
+        private readonly IRazorLightEngine _engine;
+        public TnoRazorLightEngine()
+        {
+            _engine = new RazorLightEngineBuilder()
+                .UseEmbeddedResourcesProject(typeof(TemplateModel))
+                .Build();
+        }
+        public async Task<string> CompileRenderStringAsync<T>(string key, string content, T model)
```

Comment:
You use a generic type here, but you build the engine with a fixed type.

Reply:
Updated, thanks